go/vt/vttablet/onlineddl/executor.go

```diff
@@ -2275,8 +2313,33 @@ func (e *Executor) generateSwapTablesStatement(ctx context.Context, tableName1,
 	return parsed.Query, swapTableName, nil
 }
 
-func (e *Executor) executeAlterViewOnline(ctx context.Context, onlineDDL *schema.OnlineDDL) (err error) {
+// renameTableIfApplicable renames a table, assuming it exists and that the target does not exist.
+func (e *Executor) renameTableIfApplicable(ctx context.Context, fromTableName, toTableName string) (attemptMade bool, err error) {
```

Comment:
I see that the `attemptMade` is discarded by the callers. Was it just added for debugging?